emily/handler/src/database/accessors.rs

```diff
@@ -138,6 +140,38 @@ pub async fn get_deposit_entries_for_transaction(
     .await
 }
 
+/// Pulls in a deposit entry and then updates it, retrying the specified number
+/// of times when there's a version conflict.
+pub async fn pull_and_update_deposit_with_retry(
```

Comment:
was there no way to make a generic `pull_and_update_with_retry`? not worth changing right now, but if there is we should aim for that in future changes to this area imo.

Reply:
I agree, I was just trying to do the fix quickly, and there are other places where I needed to do these as one offs. I'll open a tech debt issue for it.

Reply:
https://github.com/stacks-network/sbtc/issues/792

Reply:
:pray: